pkgs/development/interpreters/python/hooks/fix-python-deps.sh

```diff
@@ -0,0 +1,76 @@
+# shellcheck shell=bash
+
+# Setup hook that fixes Python dependencies versions.
+#
+# Example usage in a derivation:
+#
+#   { …, pythonPackages, … }:
+#
+#   pythonPackages.buildPythonPackage {
+#     …
+#     nativeBuildInputs = [ pythonPackages.fixPythonDepsHook ];
+#
+#     # Already the default, only change if necessary
+#     requirementsTxt = "requirements.txt";
+#
+#     # This will remove the upperbound version of the package
+#     # e.g.: abc>1<=2 -> abc>1
+#     relaxPythonDeps = [ "abc" ];
+#     # This will remove all dependencies restrictions
+#     # e.g.: cde>1<=2 -> cde
+#     unlockPythonDeps = [ "cde" ];
+#     # This will remove the dependency
+#     # e.g.: fgh>1<=2 -> <nothing>
+#     removePythonDeps = [ "cde" ];
+#     …
+#   }
+#
+# This hook will copy files which are either given by full path
+# or all '*.desktop' files placed inside the 'share/applications'
+# folder of each `desktopItems` argument.
+
+declare -a relaxPythonDeps
+declare -a unlockPythonDeps
+declare -a removePythonDeps
+
+requirementsTxt="${requirementsTxt:-requirements.txt}"
+
+relaxPythonDepsHook() {
+    echo "Executing relaxPythonDepsHook"
+
+    if [[ -z "$relaxPythonDeps" ]]; then return; fi
+    for dep in ${relaxPythonDeps[@]}; do
+        sed -r "s|${dep}([>=0-9.,]*)[ <=0-9.]*|${dep}\1|" -i "$requirementsTxt"
+    done
+}
+
+unlockPythonDepsHook() {
+    echo "Executing unlockPythonDepsHook"
+
+    if [[ -z "$unlockPythonDeps" ]]; then return; fi
+    for dep in ${unlockPythonDeps[@]}; do
+        sed -r "s|(${dep})[ ><=0-9.,]*|\1|" -i "$requirementsTxt"
+    done
+}
+
+removePythonDepsHook() {
+    echo "Executing removePythonDepsHook"
+
+    if [[ -z "$removePythonDeps" ]]; then return; fi
+    for dep in ${removePythonDeps[@]}; do
+        sed -r "s|${dep}[ ><=0-9.,]*||" -i "$requirementsTxt"
+    done
+}
+
+fixPythonDepsHook() {
+    relaxPythonDepsHook
+    unlockPythonDepsHook
+    removePythonDepsHook
+
+    if (( "${NIX_DEBUG:-0}" >= 1 )); then
```

Comment:
```suggestion
    if (( "${NIX_DEBUG:-0}" >= 7 )); then
```
Or are the other python hooks like this?

Reply:
Done.